src/client/helpers/apiHelpers.js

```diff
@@ -109,3 +109,40 @@ export const isWalletTransaction = actionType =>
   actionType === accountHistoryConstants.TRANSFER_TO_SAVINGS ||
   actionType === accountHistoryConstants.DELEGATE_VESTING_SHARES ||
   actionType === accountHistoryConstants.CLAIM_REWARD_BALANCE;
+
+export const getLookupAccountNames = (name, limit = 5) =>
+  SteemAPI.sendAsync('lookup_accounts', [name, limit]);
+
+export const getAllSearchResultPages = search => Promise.all([
+  fetch(
```

Comment:
I think it would be nice to have loop that would create array of promises instead of hardcoding those.